deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -751,21 +751,18 @@ spec:
                         properties:
                           clusterName:
                             type: string
-                          host:
-                            type: string
                           name:
                             type: string
                           namespace:
                             type: string
-                          port:
-                            format: int32
-                            type: integer
                           secretName:
                             type: string
                           url:
+                            pattern: ^(kubernetes|minikube|openshift|xsite):\/\/(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])(:[0-9]+)?$
```

Comment:
`_` is no supported. is it intentional?
the following example won't match:
```
minikube://my_host.com
```

Reply:
Yes, thanks @pruivo
Will change regex for this case